Cargo.toml

```diff
@@ -1,23 +1,25 @@
 [workspace]
 
 members = [
-    "crates/acvm_backend_barretenberg",
-    "crates/noirc_evaluator",
-    "crates/noirc_frontend",
-    "crates/noirc_errors",
-    "crates/noirc_driver",
-    "crates/noirc_printable_type",
-    "crates/nargo",
-    "crates/nargo_cli",
-    "crates/nargo_toml",
-    "crates/fm",
-    "crates/arena",
-    "crates/noirc_abi",
-    "crates/noirc_abi_wasm",
-    "crates/iter-extended",
-    "crates/wasm",
+    "compiler/noirc_evaluator",
+    "compiler/noirc_frontend",
+    "compiler/noirc_errors",
+    "compiler/noirc_driver",
+    "compiler/noirc_printable_type",
+    "compiler/fm",
+    "compiler/wasm",
```

Comment:
This being called wasm is a bit ambiguous, I think the closest thing to the truth would be `noirc_driver_wasm`

Reply:
Yeah, this name has been bugging me for a while. `noirc_driver_wasm` isn't really true as it's also optimizing the ACIR which is a non-compiler action.

Reply:
_sweats a bit_ ... `nargo_wasm`

Reply:
Going to follow up with this in another PR